stable/spark-history-server/templates/_helpers.tpl

```diff
@@ -41,3 +41,19 @@ Create the name of the service account to use
     {{ default "default" .Values.serviceAccount.name }}
 {{- end -}}
 {{- end -}}
+
+{{- define "spark-history-server.clusterRole" -}}
+{{- if .Values.rbac.create -}}
```

Comment:
The if statement is unnecessary since the else case never applies. But my question was why you want to have this configurable at all.

Reply:
@unguiculus Now I've made them non-configurable. Is the PR looking good this time? Can you help me merge it?